docs/rest_api_2.rst

```diff
@@ -0,0 +1,4156 @@
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Use the dn property from the object browser for input parameter “lif”                                                       |                                                                                                                                                         |
++-------------+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| lIfCtxDn    | - This value is null for PHYSICAL type of ADC devices                                                                          | uni/tn-Demo/GraphInst\_C-[uni/tn-Demo/brc-2ARMVE-34-35]-G-[uni/tn-Demo/AbsGraph-2ARMVE-34\_35]-S-[uni/tn-Demo]/NodeInst-N1/LegVNode-0/EPgDef-consumer   |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | - This value can be retrieved from visore for VIRTUAL ADC Devices:                                                             |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device> → Cluster Interfaces→ <Your Interface of Interest>   |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”.                                           |                                                                                                                                                         |
```

Comment:
remove "on"s and bold instead of quotes

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)